packages/core/src/useTitleContext.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import { SharedScreenNavigationOptions, useIsFocused } from './index';
+import NavigationDocumentTitleContext from './NavigationDocumentTitleContext';
+
+let timeout: NodeJS.Timeout | number | undefined;
+
+let nonNavigationTitle: string | undefined;
+
+let recentSetter: string | undefined;
+
+function debounceSetTitle(title: string | undefined, key: string) {
```

Comment:
Is debounce needed? I guess it might screw up SEO stuff.

Reply:
I've just tested and seems to be not needed